### DISCUSSION TITLE
Sync Workflow Creation

### DISCUSSION THREAD

**Discussion by hjjobs** [Netflix/conductor#3332] [General]
Hey Folks,

We have a use case where we want to create Sync Workflow, where we have a set of tasks which needs to run all in sync when we create a workflow and give the response of workflow failure/success in synchronous manner. I know one way is that i can trigger the workflow and keep polling the workflow status, to know about the success/failure. But Could you guys please help with a better way to achieve this using Conductor ?

**Comment by v1r3n** [comment]
Hi @hjjobs we made this simpler so you don't have to keep polling with orkes-conductor.
you can take a look at the example here: https://github.com/conductor-sdk/java-sdk-examples

we can discuss more if required.
